### PR TITLE
Feature refractor with api and styling

### PR DESCRIPTION
**Pull request by IsmailChangezi**
## Description
- Use Axios Library
- Use React icon Libraries
- Use API for data transfer
- Add some styles
- Fix Linter errors

### CODE REVIEW

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @IsmailChangezi   👋

Your project is complete! ✔️✔ , I loved your work, it looks great 🔥🔥🔥🔥 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors

- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <input
+          className="inp-author"
+          type="text"
+          value={author}
+          onChange={(e) => setAuthor(e.target.value)}
+          placeholder="Book author"
+        />
+        <button className="add-btn" type="submit">
+          ADD BOOK
+        </button>
```
Comment (AmaduKamara) [inline src/components/AddBook.js]: - [ ] Please align the input fields and the button to stretch to the same width as the book cards so your design finally matches the given design. 👍🏽

**Your Design**
Screenshot 2023-04-08 at 11 27 12 AM

**The Original Design**
Screenshot 2023-04-08 at 11 22 30 AM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @IsmailChangezi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights:

✅ You used Gitflow correctly
✅ Linter checks passed 
✅ App works fine as expected 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: var(--white-two);
+}
```
Comment (franclobo) [inline src/App.css:280]: - **[OPTIONAL]** Well done implementing the required changes, but it doesn't look like the original design yet. Please 🙏🏽 align the add button with the inputs on the form.

image

**Review by franclobo — APPROVED** [review]
Hi, @IsmailChangezi, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️
- App works as expected ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**